src/remote/action.go

```diff
@@ -474,7 +474,7 @@ func (c *Client) verifyActionResult(target *core.BuildTarget, command *pb.Comman
 
 // uploadLocalTarget uploads the outputs of a target that was built locally.
 func (c *Client) uploadLocalTarget(target *core.BuildTarget) error {
```

Comment:
I had to update this stuff because sigstore requires gRPC for the kms API, and that had a breaking change. Damn google and their disregard for Go packaging standards.